ruby_challenges.rb

```diff
@@ -1,45 +1,76 @@
 # ASSESSMENT 5: Ruby Coding Practical Questions
 
-# --------------------1) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data about each bike object.
-
-
-
-
+# 1) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. .
+
+class Bike
+    attr_accessor :model, :frame_size
+    def initialize (model, frame_size)
+        @model = model
+        @wheels = 2
+        @frame_size = frame_size
+        # 3) Add a speedometer to the Bike class. The speed should be initialized at 0.
+        @speedometer = 0
+    end
+
+    #Create a get_info method that returns a sentence with all the data about each bike object
+    def get_info
+        "The #{model} bike has #{@wheels} wheels and a #{frame_size} frame."
```

Comment:
interesting choice with using the class getter methods. Consider why using the getter method would be more useful than the instance variable